src/uu/tail/src/tail.rs

```diff
@@ -188,8 +190,9 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
             if path.is_dir() {
                 continue;
             }
+            let md = metadata(path).unwrap();
             let mut file = File::open(&path).unwrap();
-            if is_seekable(&mut file) {
+            if is_seekable(&mut file) && md.blocks() > 0 {
```

Comment:
This has to be guarded so the extra check only happens on Unix. Ideally the `.metadata()` call would also only happen on Unix.

Reply:
Yes, I know, I had it on draft to fix this.